app/src/main/res/values/strings.xml

```diff
@@ -533,36 +533,31 @@
+        <item>Can I create a new text file using the app?#####No, you cannot. You can only convert an already created text file to PDF.</item>
+        <item>Is there a limit on number of images I can add at once to create a PDF?#####No, there are no restrictions.</item>
+        <item>Is there a limit to the image size I can add?#####No, there are no limits. However, we advise to restrict the size to less than 10MB. The conversion of multiple large image files take upwards of 10 seconds.</item>
+        <item>Can I insert captions in the image after adding?#####No, you cannot.</item>
+        <item>Can I read PDF files using the app?#####No, you cannot as of now. This app does not work as a PDF viewer.</item>
+        <item>I want to access the PDF files I created externally outside the software, where are the pdf files saved?#####The PDFfiles files created with the app are automatically saved in the folder named PDFfiles on your device storage.</item>
+        <item>How to change the default storgae location?#####To change the default storage location, \n1) Click the Hamburger icon (three horizontal line) at the top-left corner. \n2) Scroll down and click Settings. \n3) In the settings page, click the field just beneath the text Change storage location. \n4) Open the preferred save location and click the Select icon at the top. \nOnce changed, the files created using the app will be stored in the new location.</item>
+        <item>How can I share the PDF files that are edited/created using the app?#####You can access and share the PDF files which you have edited/created directly from the app. To share the PDF files, \n1) Click the View Files option on the home page to view all the PDF files you have created. \n2) Select the file(s) you want to share using the checkbox. \n3) Next, Click the More option (three vertical dots) at the top-right and click Share. \n4) Select the desired method to share the files.</item>
+        <item>Can I recover the forgotten password of a PDF file using the app?#####No, a lost password cannot be recovered.</item>
+        <item>Can I change the password of a PDF file?#####Yes, but only if you remove the existing password. Once the password is removed, use the Remove Passowrd option in the home page to change the password.</item>
+        <item>Do you collect any personal data when using the app?#####No, we do not collect any personal data.</item>
+        <item>Can I compress the images before creating a PDF?#####Yes, you can. To compress the images, \n1) Click the Images to PDF option on the homepage. \n2) Select the images using the SELECT IMAGES button. \n3) Next, select the Image compression option to compress the images.</item>
+        <item>What is the format of the images extracted from a PDF?#####The images extracted from a PDF will be in .png format.</item>
+        <item>Can I add files from cloud/online storage?#####No, you cannot as of now.</item>
```

Comment:
`No, files from cloud storage cannot be selected at the moment but we are planning on adding it soon!`

Reply:
Got it. Will update this.

Reply:
you should mark the conversation as resolved, after you have committed a change to fix the particular review comment.

Reply:
Got it. Thanks.